.gopherci.yml

```diff
@@ -0,0 +1,4 @@
+# xorg-dev and libgl1-mesa-dev are cgo dependencies for glfw (imported by dumpglfw3joysticks).
+apt_packages:
+    - xorg-dev
+    - libgl1-mesa-dev
```

Comment:
Can you swap the order to have them consistent with .travis.yml file, and also use 2 spaces for indent, again for consistency with .travis.yml.